### DISCUSSION TITLE
Panel Zoom help

### DISCUSSION THREAD

**Discussion by j-d-rutherford** [koreader/koreader#7338] [Q&A]
I am using KOReader on a 6" Kobo Tablet;

I would like to know how to properly use Panel Zoom.  Is it possible to specify a page layout (IE top-> bottom, RTL (manga)) and then use the screen tap to move from one zoomed panel to the next? The page-turn tap when zoomed should continue through the page to the next panel, but instead I need to long-press on the image, read it, tap to bring up menu, close the image, and long press the next image... etc.

I feel like I am using this feature incorrectly.

**Comment by poire-z** [comment]
No, you're just not using the right feature :)
Disable panel zoom, what you want is #6885, and is in the bottom pannel: 
<kbd>![image](https://user-images.githubusercontent.com/24273478/108788835-2fe59600-7579-11eb-9ef5-8e199a4e20fc.png)</kbd>
